docs/dashboards/your-first-dashboard.md

```diff
@@ -19,26 +19,28 @@ Thus, we have to import the main dashboard package, the Highcharts library and a
     ```
 
 3. In order to properly display the whole dashboard, we need some styling. You can get the basic one by importing the modules to the CSS file as shown below.
+If you want to use Highcharts charts, you should also import the charts styles, because the [styled mode](https://www.highcharts.com/docs/chart-design-and-style/custom-themes-in-styled-mode) is enabled by default in Dashboards.
```

Comment:
In point 2 it's worth mentioning, with a URL, other ways you can install the Dashboards.